Source/Core/VideoCommon/FreeLookCamera.cpp

```diff
@@ -319,10 +310,7 @@ void FreeLookCamera::DoState(PointerWrap& p)
   else
   {
     const auto old_type = m_current_type;
```

Comment:
There's also a warning (on the `const std::string old_type_name = old_type ? to_string(*old_type) : ""`) on the [ubuntu builder](https://dolphin.ci/#/builders/19/builds/3724) that `old_type` may be uninitialized.  This [might be a GCC bug](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80635), but it's odd.  (It also isn't new to this PR).  Possibly switching to an `EnumFormatter` here would work, though I'm not sure.  It's probably fine to just ignore this.